client/landing/stepper/declarative-flow/helpers/get-step-old-slug.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * This can be used when renaming a step. Simply add a map entry with the new step slug and the old step slug and Stepper will fire `calypso_signup_step_start` events for both slugs. This ensures that funnels with the old slug will still work.
+ */
+export const getStepOldSlug = ( stepSlug: string ): string | undefined => {
+	const stepSlugMap: Record< string, string > = {
+		'create-site': 'site-creation-step',
+	};
+
+	return stepSlugMap[ stepSlug ];
+};
```

Comment:
Maybe just make this a map?
```suggestion
export const oldStepSlugs = {
	'create-site': 'site-creation-step',
} as const;
```

Reply:
I just moved this for the PR -- I think we should refactor in a follow-up, as this is called from at least two places.